common/v2/features/Settings/components/GeneralSettings.tsx

```diff
@@ -73,17 +73,17 @@ interface SettingsProps {
 }
 
 const timerOptions = [
-  { name: '1 Minutes', value: '60000' },
-  { name: '3 Minutes', value: '180000' },
-  { name: '5 Minutes', value: '300000' },
-  { name: '10 Minutes', value: '600000' },
-  { name: '15 Minutes', value: '900000' },
-  { name: '30 Minutes', value: '1800000' },
-  { name: '45 Minutes', value: '2700000' },
-  { name: '1 Hours', value: '3600000' },
-  { name: '3 Hours', value: '10800000' },
-  { name: '6 Hours', value: '21600000' },
-  { name: '12 Hours', value: '43200000' }
+  { name: `1 ${translateRaw('ELAPSED_TIME_MINUTE')}`, value: '60000' },
```

Comment:
Could probably pass these numbers into the translation strings using vars. Is there a reason we wouldn't do that?